src/main/java/net/geforcemods/securitycraft/screen/EditModuleScreen.java

```diff
@@ -76,6 +91,24 @@ public boolean keyPressed(int keyCode, int scanCode, int p_keyPressed_3_)
 	{
 		if(inputField.isFocused())
 		{
+			if (keyCode == GLFW.GLFW_KEY_BACKSPACE)
+			{
+				for(int i = 1; i <= ModuleItem.MAX_PLAYERS; i++)
+				{
+					if(!inputField.getText().isEmpty() && module.getTag() != null)
+						if (module.getTag().getString("Player" + i).equals(inputField.getText().substring(0, inputField.getText().length()-1)))
+						{
+						addingButton.active = false;
+						removeButton.active = !(inputField.getText().length() <= 1);
+						break;
+						}
+					if (i == ModuleItem.MAX_PLAYERS) {
+						addingButton.active = !(inputField.getText().length() <= 1);
+						removeButton.active = false;
+					}
+				}
+				if (inputField.getText().isEmpty()) return false;
```

Comment:
```suggestion

				if (inputField.getText().isEmpty())
					return false;
```

Reply:
You didn't add the new line above the if statement from this suggested change.

Reply:
Sorry, I somehow overlooked this. Also, you can't really see where a new line is added or not in the suggested change picture, as there is no horizontal scroll bar and longer lines stretch over multiple lines, so it gets unclear what line is new and what not.